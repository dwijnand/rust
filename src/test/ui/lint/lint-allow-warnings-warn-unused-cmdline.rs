// Tests -W unused-variables overrides and triggers the warning

// compile-pass
// compile-flags: -A warnings -W unused-variables

fn main() {
    let x = 3;
}
