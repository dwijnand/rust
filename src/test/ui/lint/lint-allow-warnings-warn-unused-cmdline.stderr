warning: unused variable: `x`
  --> $DIR/lint-allow-warnings-warn-unused-cmdline.rs:7:9
   |
LL |     let x = 3;
   |         ^ help: consider using `_x` instead
   |
   = note: requested on the command line with `-W unused-variables`

